ruby-challenges.rb

```diff
@@ -2,33 +2,67 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
-
+# pueudo code #
+# set up an def and end def : detEvenOdd fuction to determine
+# use "number" for the values \.
+#  create a conditional .even? if and else 
```

Comment:
use the pseudocode to explain how you will manipulate the data 

Example:
```ruby
# set up an def and end def : detEvenOdd fuction to determine
# use "number" for the values \.
#  create a conditional .even? if and else 
```
with explanation
```ruby
declare a method named detEvenOdd 
input: integer, parameter will be called number 
output: string that states whether the integer is even or odd 
if given 7 return '7 is odd' 

process: Use conditionals (if/else) to evaluate if the integer is even (.even?) 
If true , return that the integer is even 
If false, return that the integer is odd
```